src/main/java/com/puppycrawl/tools/checkstyle/utils/ModuleReflectionUtils.java

```diff
@@ -76,17 +78,36 @@ public static boolean isCheckstyleModule(Class<?> clazz) {
                     || isFilterModule(clazz)
                     || isFileFilterModule(clazz)
                     || isTreeWalkerFilterModule(clazz)
+                    || isAuditListener(clazz)
```

Comment:
Listener is not a module ..... smth is broken in terminology .... but most likely we do smth wrong here, and fix should be done not in this code.

lets come back to original problem. Why you need to load custom Listener by checkstyle code ?